File: 1366.py
while True:
    n = int(input())
    if n == 0:
        break

    pares = 0
    for i in range(n):
        ci, vi = map(int, input().split())
        pares += int(vi // 2)

    pares //= 2
    print(pares)

File: 1366.js
const fs = require('fs');
const data = fs.readFileSync(0, 'utf8').trim().split(/\s+/).map(Number);

let idx = 0;

while (true) {
    let n = data[idx++];
    if (n === 0) break;

    let pares = 0;
    for (let i = 0; i < n; i++) {
        let ci = data[idx++];
        let vi = data[idx++];

        pares += Math.floor(vi / 2);
    }

    console.log(Math.floor(pares /= 2));
}   